resources/tr/cemu.po

```diff
@@ -2,156 +2,763 @@ msgid ""
+msgid "No updates available."
+msgstr "Güncelleme bulunamadı."
+
+#: DownloadGraphicPacksWindow.cpp:224
+msgid ""
+"Updated graphic packs are available. Do you want to download and install "
+"them?"
+msgstr "Güncel grafik paketleri bulundu. Güncellemek ister misiniz?"
+
+#: DownloadGraphicPacksWindow.cpp:333
+msgid "Checking version..."
+msgstr "Sürüm kontrol ediliyor..."
+
+#: DownloadGraphicPacksWindow.cpp:340 updateWindow.cpp:24
+msgid "Cancel"
+msgstr "İptal"
+
+#: DownloadGraphicPacksWindow.cpp:409
+msgid "Downloading graphic packs..."
+msgstr "Grafik paketleri indiriliyor..."
+
+#: DownloadGraphicPacksWindow.cpp:413
+msgid "Extracting..."
+msgstr "Çıkarılıyor..."
+
+#: DumpCtrl.cpp:172 RegisterCtrl.cpp:205 RegisterCtrl.cpp:224
+#: RegisterCtrl.cpp:238 RegisterWindow.cpp:258 RegisterWindow.cpp:274
+#: RegisterWindow.cpp:290
+msgid "Enter a new value."
+msgstr "Yeni bir değer girin."
+
+#: DumpCtrl.cpp:235 InputSettings.cpp:101 InputSettings.cpp:496
+#: InputSettings.cpp:502 InputSettings.cpp:571 InputSettings.cpp:585
+#: InputSettings.cpp:591 InputSettings.cpp:596 InputSettings.cpp:609
+#: MainWindow.cpp:2328 MainWindow.cpp:2332 MainWindow.cpp:2336
+#: MainWindow.cpp:2340 MainWindow.cpp:2345 MainWindow.cpp:2353
+#: toolMemorySearcher.cpp:164
+msgid "Error"
+msgstr "Hata"
+
+#: GameProfileWindow.cpp:16 GameProfileWindow.cpp:239
+msgid "Edit game profile"
+msgstr "&Oyun profili düzenle"
+
+#: GameProfileWindow.cpp:33 GameProfileWindow.cpp:85 GeneralSettings2.cpp:158
+#: GeneralSettings2.cpp:166 GeneralSettings2.cpp:357
+msgid "General"
+msgstr "Genel"
+
+#: GameProfileWindow.cpp:36
+msgid "Load shared libraries"
+msgstr "Paylaşılan kütüphaneleri yükle"
+
+#: GameProfileWindow.cpp:37
+msgid ""
+"EXPERT OPTION\n"
+"This option will load libraries from the cafeLibs directory"
+msgstr ""
+"UZMAN SEÇENEĞİ\n"
+"Bu ayar cafeLibs klasöründeki kütüphaneleri yükler"
+
+#: GameProfileWindow.cpp:44
+msgid "CPU"
+msgstr "CPU"
+
+#: GameProfileWindow.cpp:51
+msgid "Mode"
+msgstr "Mod"
+
+#: GameProfileWindow.cpp:53
+msgid "Singlecore-Interpreter"
+msgstr "TekÇekirdek-Yorumlayıcı"
+
+#: GameProfileWindow.cpp:53
+msgid "Singlecore-Recompiler"
+msgstr "TekÇekirdek-Derleyici"
+
+#: GameProfileWindow.cpp:53
+msgid "Dualcore-Recompiler"
+msgstr "ÇiftÇekirdek-Derleyici"
+
+#: GameProfileWindow.cpp:53
+msgid "Triplecore-Recompiler"
+msgstr "ÜçÇekircdek-Derleyici"
+
+#: GameProfileWindow.cpp:56
+msgid "Set the CPU emulation mode"
+msgstr "CPU taklit emülasyon seç"
+
+#: GameProfileWindow.cpp:59
+msgid "Thread quantum"
+msgstr "İş Parçacığı Kuantumu"
+
+#: GameProfileWindow.cpp:69
+msgid ""
+"EXPERT OPTION\n"
+"Set the maximum thread slice runtime (in virtual cycles)"
+msgstr ""
+"UZMAN SEÇENEĞİ\n"
+"Maksimum iş parçacığı dilimi çalışma zamanını ayarla (sanal döngülerde)"
+
+#: GameProfileWindow.cpp:72
+msgid "cycles"
+msgstr "döngüler"
+
+#: GameProfileWindow.cpp:95
+msgid "Extended texture readback"
+msgstr "Genişletilmiş doku okuma"
```

Comment:
Searching "Genişletilmiş doku okuma" on Google I didn't found anything related to 3D graphics. Maybe "texture readback" should be kept untranslated (unless your language has a very specific translation for "texture").